pages/_document.tsx

```diff
@@ -1,12 +1,6 @@
-import NextDocument, {
-  DocumentContext,
-  DocumentInitialProps,
-  Html,
-  Head,
-  Main,
-  NextScript
-} from "next/document";
-import { ServerStyleSheet } from "styled-components";
+import NextDocument, { DocumentContext, DocumentInitialProps, Html, Head, Main, NextScript } from 'next/document';
+import { ServerStyleSheet } from 'styled-components';
+import * as React from 'react';
```

Comment:
These changes are not what you intended, they are by your editor's auto-formatting, right?

Seems not related on this pull request. Could you restore these changes?

Reply:
```diff
+ import * as React from 'react';
```
I needed it for compile error. Is it ok?

Reply:
Yep, thanks for the change!